src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
```

Comment:
Please make code more readable

```suggestion
  const children = onlyWithSon
    ? people.filter((p) => {
        return (p.sex === 'm'
          && people.some((m) => {
            return m.name === p.mother;
          })
        )
      })
    : people.filter((p) => {
        return people.some((m) => {
          return m.name === p.mother;
        })
      });
```